appf.adoc

```diff
@@ -477,6 +525,9 @@ Domain: +
 // The cleanest-looking, which sometimes works for asciidoctor-pdf:
      **`-90.0 \<= latitude_of_projection_origin \<= 90.0`**
 
+| **`refinement_level`** | N
+| DGGS numerical order of a discrete global grid in the tessellation sequence. See "4.37. refinement level" in <<OGC_DGGS>>.
```

Comment:
After the conversation about relating DGGRS -> DGGS -> DGGH, I am not sure about using an abbreviation in this text. Can we just call it the level within the cell hierarchy while keeping the reference for providing details in the DGG(R)S context?